packages/hardware-trezor/test/transformers/withdrawals.test.ts

```diff
@@ -0,0 +1,64 @@
+import { Cardano } from '@cardano-sdk/core';
+import { CardanoKeyConst, util } from '@cardano-sdk/key-management';
+import {
+  contextWithKnownAddresses,
+  contextWithoutKnownAddresses,
+  coreWithdrawalWithKeyHashCredential,
+  coreWithdrawalWithScriptHashCredential,
+  stakeKeyHash,
+  stakeScriptHash
+} from '../testData';
+import { mapWithdrawals, toWithdrawal } from '../../src/transformers';
+
+describe('withdrawals', () => {
+  describe('mapWithdrawals', () => {
+    it('return undefined if given an undefined object as withdrawal', async () => {
+      const withdrawals: Cardano.Withdrawal[] | undefined = undefined;
+      const txIns = mapWithdrawals(withdrawals, contextWithKnownAddresses);
+      expect(txIns).toEqual(undefined);
+    });
+
+    it('can map a a set of withdrawals', async () => {
+      const withdrawals = await mapWithdrawals(
+        [coreWithdrawalWithKeyHashCredential, coreWithdrawalWithKeyHashCredential, coreWithdrawalWithKeyHashCredential],
+        contextWithKnownAddresses
+      );
+
+      expect(withdrawals!.length).toEqual(3);
+
+      for (const withdrawal of withdrawals!) {
```

Comment:
let's replace this with an array comparison

Reply:
✅